Common/Sources/Extensions/String.swift

```diff
@@ -79,6 +79,30 @@ public extension String {
         }
     }
 
+    /// Format an amount to a currency format with currency code
+    ///
+    /// - Parameter currencyCode: the currency code
+    /// - Returns: a formatted String amount with currency code
+    func formatWithActualDecimal(with currencyCode: String?) -> String {
+        if !self.isEmpty {
+            let currencyFormatter = NumberFormatter()
+            let locale = Locale(identifier: Locale.preferredLanguages[0])
+            let localizedDecimalSeparator: Character = locale.decimalSeparator?.first ?? "."
+            let components = self.split(separator: localizedDecimalSeparator)
```

Comment:
we don't use locale and localization in the Number formatters.

Reply:
Code has been modified.